InternxtDesktop/Services/BackupsService.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  BackupsService.swift
```

Comment:
We will probably have another target for the Backups engine, so I suggest you to move the BackupsService to the Shared group, so we can use it from everywhere